src/util/popup.ts

```diff
@@ -18,7 +18,7 @@ export class PopupService<T> {
     this._windowFactory = componentFactoryResolver.resolveComponentFactory<T>(type);
   }
 
-  open(content: string | TemplateRef<any>): ComponentRef<T> {
+  open(content: string | TemplateRef<any> = ''): ComponentRef<T> {
```

Comment:
I would rather check if it is null / undefined in the `_getContentNodes` (and return `[]` in this case) to avoid unnecessary text nodes creation.